include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -452,11 +448,7 @@ struct __sort_fn
         const auto __dispatch_tag = oneapi::dpl::__ranges::__select_backend(__exec);
         return oneapi::dpl::__internal::__ranges::__pattern_sort_ranges(__dispatch_tag,
             std::forward<_ExecutionPolicy>(__exec), std::forward<_R>(__r), __comp, __proj,
-#if _ONEDPL_STD_RANGES_ALGO_CPP_FUN
-            [](auto&&... __args) { std::ranges::sort(std::forward<decltype(__args)>(__args)...); }
-#else
-            std::ranges::sort
-#endif
+            [](auto&&... __args) { return std::ranges::sort(std::forward<decltype(__args)>(__args)...); }
```

Comment:
Could you apply clang-format? [clang-format.patch](https://github.com/user-attachments/files/17798400/clang-format.patch)